tools/footprint/footprint.xml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<tool id="footprint" name="footprint" version="1.0.0">
+    <requirements>
+        <requirement type="package" version="1.0.0">footprint</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+            find_footprints.sh
+
+            '$bam_file'
+
+            '$chrom_sizes'
+
+            '$motif_coords'
+
+            ## genome source
+            #if $refGenomeSource.genomeSource == "history":
+                '$refGenomeSource.ownFile'
+            #else
+                '$refGenomeSource.builtin.fields.path'
+            #end if
+
+            $factor_name
+
+            '$bias_file'
+
+            '$peak_file'
+
+            $no_of_components
+
+            $background
+
+            $fixed_bg
+
+            &&
+
+            input_dir=\$(python -c "import os; print(os.path.dirname('$bam_file'))")
+
+            &&
+            mv \$(find \$input_dir -name "*.PARAM") PARAM
+
+            &&
+            mv \$(find \$input_dir -name "*.RESULTS") RESULTS
+
+            &&
+            mv \$(find \$input_dir  -name "*.plot2.png") plot2.png
```

Comment:
`mv $input_dir/*.plot2.png plot2.png` or is this a wired nested folder hierarchy?